domains.txt

```diff
@@ -1,3 +1 @@
-firebase.flutter.dev firebaseextended/flutterfire
-melos.invertase.dev invertase/melos
-ehesp.invertase.io ehesp/testing
+melos.invertase.io invertase/melos
```

Comment:
Shouldn't this be the .dev domain?

Reply:
Yeah was for testing mainly. Couldn't actually find the .dev domain, but thought I'd leave it until the proxy is ready.